packages/superset-ui-plugin-chart-icicle-event/src/IcicleEventChart.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component, createRef } from 'react';
```

Comment:
Add file headers. Comment applies to the other files. This should make one of the build pre-checks pass.

Reply:
Added the file headers to all of the files but the checks are failing due to a timeout

I believe this PR is related and will be a fix for the Netlify checks: 
https://github.com/apache-superset/superset-ui-plugins/pull/160